images/initials.go

```diff
@@ -0,0 +1,84 @@
+package images
+
+import (
+	"bytes"
+	"image/color"
+	"image/png"
+	"io/ioutil"
+	"strings"
+
+	"github.com/fogleman/gg"
+	"golang.org/x/image/font"
+	"golang.org/x/image/font/opentype"
+)
+
+type RGBA struct {
+	R, G, B, A float64
+}
+
+var parsedFont *opentype.Font = nil
```

Comment:
If you are relying on a font global perhaps the font should be loaded as part of the package `init()` function.

There seems to be a lot of work-around work to ensure a dynamic parameter `fontFile` isn't parsed more than once, and there is a presumption in the code that only one fontFile will be used in the application lifecycle.

Perhaps instead of passing the font file into `status-go` as a parameter, add the file as a binary asset? Then you can reference the font bytes directly without the need for file-paths.

Reply:
I can help you with this if you need any assistance.

Reply:
That would be great. There's also a hack on the mobile side to move the file to a location on android on startup in order to get the path that can be accessed by status-go.
@cammellos we talked about this and you prefer to pass the font file path from mobile, any inputs?